src/sagas.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * Deprecation Notice
+ */
+
```

Comment:
Should there be another line here with content for this reminder?

Reply:
This whole file is going away since redux-saga is no longer recommended!